ksql-parser/src/main/java/io/confluent/ksql/parser/tree/AbstractStreamCreateStatement.java

```diff
@@ -31,23 +30,38 @@
   private final QualifiedName name;
   private final ImmutableList<TableElement> elements;
   private final boolean notExists;
-  private final ImmutableMap<String, Literal> properties;
+  private final StatementProperties properties;
 
   AbstractStreamCreateStatement(
       final Optional<NodeLocation> location,
       final QualifiedName name,
       final List<TableElement> elements,
       final boolean notExists,
       final Map<String, Literal> properties
+  ) {
```

Comment:
can we not just drop the old constructor?

Reply:
I suppose we can :)